pkg/reconciler/v1alpha1/route/route_test.go

```diff
@@ -184,6 +184,13 @@ func newTestSetup(t *testing.T, configs ...*corev1.ConfigMap) (
 				prodDomainSuffix:    "selector:\n  app: prod",
 			},
 		},
+		{
```

Comment:
This table test could be using:
```
{{
   ...
}, {
   ...
}}
```
style

Reply:
Done